catalog/transformer/catalog-transformer-service-atom/src/test/java/ddf/catalog/transformer/response/query/atom/TestAtomTransformer.java

```diff
@@ -187,34 +187,41 @@ public void testNoDdfConfiguration()
         // given
         MetacardTransformer metacardTransformer = getXmlMetacardTransformerStub();
 
-        SystemInfo systemInfo = mock(SystemInfo.class);
-        when(systemInfo.getSiteName()).thenReturn("");
+        String currentSiteName = System.getProperty(SystemInfo.SITE_NAME);
 
-        AtomTransformer transformer = new AtomTransformer(systemInfo);
-        transformer.setMetacardTransformer(metacardTransformer);
-        SourceResponse response = getSourceResponseStub(SAMPLE_ID, null);
-
-        // when
-        BinaryContent binaryContent = transformer.transform(response, null);
-
-        // then
-        assertThat(binaryContent.getMimeType(), is(AtomTransformer.MIME_TYPE));
+        try {
+            System.setProperty(SystemInfo.SITE_NAME, "");
```

Comment:
This is the concern I have with this change, i.e., the tests cannot mock `SystemInfo` and `SystemBaseUrl` anymore and have to know about their implementation to actually have them return specific values.